index.bs

```diff
@@ -1962,24 +1962,55 @@ Note: While APIs should not
 expose a full list of devices in an [=implementation-defined=] order,
 they may need to for web compatibility reasons.
 
-<h3 id="wrapper-apis">Native APIs don't typically translate well to the web</h3>
+<h3 id="usecase-oriented-apis">Design based on functionality, not the underlying API or hardware</h3>
 
+Try to design new native capabilities being brought to the web based on functionality.
```

Comment:
I am not a fan of the "try to" Just say "Design"

Reply:
based on -> around their functionality

Reply:
```suggestion
Design new native capabilities being brought to the web based on functionality.
```